src/Seksjon/Utdanning.elm

```diff
@@ -0,0 +1,193 @@
+module Seksjon.Utdanning exposing (Model, ModelInfo, Msg, Samtale(..), SamtaleStatus(..), init, meldingsLogg, update, viewUtdanning)
+
+import Cv.Utdanning as Cv exposing (Utdanning)
+import Html exposing (..)
+import Html.Attributes exposing (..)
+import Html.Events exposing (..)
+import Melding exposing (Melding(..))
+import MeldingsLogg exposing (MeldingsLogg)
+import Personalia exposing (Personalia)
+import Skjema.Utdanning exposing (UtdanningSkjema)
+
+
+
+--- MODEL ---
+
+
+type Model
+    = Model ModelInfo
+
+
+type alias ModelInfo =
+    { seksjonsMeldingsLogg : MeldingsLogg, aktivSamtale : Samtale, utdanningListe : List Utdanning, nivå : String }
+
+
+type Samtale
+    = Intro
+    | FullførtRegistrering
+    | RegistrerUtdanning
+    | RegistrerNivå NivåInfo
+    | RegistrerRetning NivåRetningInfo
+    | RegistrerSkole NivåRetningSkoleInfo
+    | RegistrerBeskrivelse NivåRetningSkoleBeskrivelseInfo
+    | RegistrerPeriode NivåRetningSkoleBeskrivelsePeriodeInfo
+    | Oppsummering NivåRetningSkoleBeskrivelsePeriodeOppsummeringInfo
+
+
+type alias NivåInfo =
+    { nivå : String }
```

Comment:
Dette er greit i første omgang, men for at dette faktisk skal funke så må jo dette være en custom type, så vi vet at vi kun har nivåer som backend skjønner. Men det kan dere ta på sikt 😊